flake.nix

```diff
@@ -105,7 +133,7 @@
             cmake
```

Comment:
Compiling in the devshell is missing the extra AOT flags var. You could either export it if you keep it as an env var, or if you add it as a CMake argument then you can wrap the CMake executable in the devshell to implicitly include it:
```
jankCmake = pkgs.writeShellScriptBin "cmake" ''
  exec ${pkgs.cmake}/bin/cmake -Djank_extra_aot_flags=... "$@"
'';

packages = [ jankCmake ... ];

```